packages/atomic/src/components/result-template-components-v1/atomic-result-multi-text/atomic-result-multi-text.tsx

```diff
@@ -0,0 +1,164 @@
+import {Component, Prop, h, State, VNode} from '@stencil/core';
+import {
+  BreadcrumbManager,
+  buildBreadcrumbManager,
+  Result,
+  ResultTemplatesHelpers,
+} from '@coveo/headless';
+import {ResultContext} from '../../result-template-components/result-template-decorators';
+import {getFieldValueCaption} from '../../../utils/field-utils';
+import {
+  Bindings,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {titleToKebab} from '../../../utils/utils';
+
+const listItemClasses = 'inline-block';
+
+/**
+ * The `atomic-result-multi-text` component renders the values of a multi-value string result field.
+ * @part result-multi-text-separator - The separator between two field values.
+ * @part result-multi-text-value - A field value.
+ * @slot result-multi-text-value-* - Allows the personalization of a particular field value based on its indexed name.
```

Comment:
As a developer using this component, what would I do if I want to personalize a field value with the following indexed name: `salty;sweet;tasty;sour`. What do I substitute the asterisk with in such a case?

Reply:
You would do the following:
```html
<atomic-result-multi-value-text field="...">
  <span slot="result-multi-value-text-value-sweet">Sweet!</span>
</atomic-result-multi-value-text>
```

I'll update the part names.

Reply:
OK, great. Thanks!

Reply:
(can't apply suggestion to outdated comment) I'd suggest we say:  "Lets you specify a custom caption value for a given part of a mutli-text field value. (e.g., if you want to use `Sweet!` as a caption value for `sweet` in `salty;sweet;sour`, you'd use  `<span slot="result-multi-value-text-value-sweet">Sweet!</span>`).

Reply:
Awesome, thanks!